cypress/support/ratings.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
I think we're probably likely to see more tests in different suites need to interact with ratings components, so it's probably good to centralize this